diff -urN b/arch/arm/boot/dts/Makefile a/arch/arm/boot/dts/Makefile
--- b/arch/arm/boot/dts/Makefile	2022-01-16 10:14:24.000000000 +0200
+++ a/arch/arm/boot/dts/Makefile	2023-08-15 20:39:57.593941237 +0300
@@ -629,7 +629,10 @@
 	imx6ull-14x14-evk.dtb \
 	imx6ull-colibri-eval-v3.dtb \
 	imx6ull-colibri-wifi-eval-v3.dtb \
+	imx6ull-ev-nano-tira.dtb \
 	imx6ull-myir-mys-6ulx-eval.dtb \
+	imx6ull-nano-cpu-v1.dtb \
+	imx6ull-nano-cpu-v2.dtb \
 	imx6ull-opos6uldev.dtb \
 	imx6ull-phytec-segin-ff-rdk-nand.dtb \
 	imx6ull-phytec-segin-ff-rdk-emmc.dtb \
